index.html

```diff
@@ -0,0 +1,177 @@
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    
+</section>
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
+         website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+      <img class="abt-me-image" src="./assets/images/Group1.png" alt="">
+   </div>
+   <hr class="line-3">
+
+   <ul class="lists">
+      <li class="Lan">Language 
+          <ul class="lan-list">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+      </li>
+        
+      <li class="Frame"> Framework
+          <ul class="Frame-list">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+          </ul>
+      </li>
+      <li class="Sk">Skills
+          <ul class="Sk-list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+      </li>
+  </ul>
+</section>
+        <form class="upper-form" action="https://formspree.io/f/xoqrrprd" method="post">
+         <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+            <input type="text" maxlength="30" required placeholder="Full Name" class="full-name" name="usrname">
+            <input type="email" required placeholder="Email" class="email" name="usremail">
+            <textarea maxlength="500" id="msg" placeholder="Enter text here" class="messages" name="usrmessages"></textarea>
+            <input type="submit" value="Get it touch" class="touch" name="usrsubmit">
+        </form>
+
+<footer class="footer-social-media">
+  <ul class="social-links">
+      <li class="socia-c"><a href="https://github.com/solog0039"><img  src="./assets/social media footer/Icon -GitHub.png" alt="Github"></a></li>
+      <li class="social-Linkdinf"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img  src="./assets/social media footer/Icon - Linkedin.png" alt="linkedin"></a></li>
+      <li class="social-c"><a href="##"><img  src="./assets/social media footer/Angelist.png" alt="tum"></a></li>
+      <li class="social-c"><a href="https://twitter.com/Solomon57320119"><img  src="./assets/social media footer/Icon - Twitter.png" alt="tiwtter"></a></li>
+      <li class="social-c"><a href="##"><img  src="./assets/social media footer/medium.png" alt="Medium"></a></li>
+  </ul>
```

Comment:
- [ ] Great work making the changes requested by the previous reviewer. During the refactoring of your code, the space between the form `button` and the `footer` was removed. Kindly put the spacing back as it will make your design identical to that of the Figma template that you chose.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/214641296-c7005845-6318-4bf4-bed8-b0e72e81640f.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/214641426-90400de9-11f8-4157-9e75-b42edffc158f.png)